src/d3a/models/myco_matcher/myco_external_matcher.py

```diff
@@ -101,30 +123,29 @@ def publish_simulation_id(self, message):
         channel = "external-myco/simulation-id/response/"
         self.myco_ext_conn.publish_json(channel, {"simulation_id": self.simulation_id})
 
-    def publish_event_tick_myco(self):
+    def event_tick(self, **kwargs):
         """Publish the tick event to the Myco client."""
-
+        is_it_time_for_external_tick = kwargs.pop("is_it_time_for_external_tick", False)
+        # If External matching is enabled, limit the number of ticks dispatched.
+        if is_it_time_for_external_tick:
```

Comment:
This should be :
```suggestion
        if not is_it_time_for_external_tick:
```
right?

Reply:
Indeed!

Reply:
Actually, this shows, that there should be a test for that. Could you please add one? Thanks a lot!